src/pyhf/tensor/pytorch_backend.py

```diff
@@ -245,11 +245,27 @@ def product(self, tensor_in, axis=None):
     def abs(self, tensor):
         return torch.abs(tensor)
 
-    def ones(self, shape):
-        return torch.ones(shape, dtype=self.dtypemap['float'])
+    def ones(self, shape, dtype="float"):
```

Comment:
Adjusting these to actually take an arg might a better idea than what we originally did with forcing `dtype=self.dtypemap["float"]` in PR #701.